# exception causing motion planning to fail
string planning_exception

# name of planning group
string planning_group

# requested motion planning target
geometry_msgs/PoseStamped requested_plan_target
